"In Suite" tests are broken

Running the test suite fails on the main branch. The test fails at the `In Suite` with a nil pointer error:
```
Running Suite: In Suite - /Users/jedge/git/simspace/gitlab-merge-request-resource/pkg/in
========================================================================================
Random Seed: 1670432625

Will run 1 of 1 specs
------------------------------
•! [PANICKED] [0.003 seconds]
In
/Users/jedge/git/simspace/gitlab-merge-request-resource/pkg/in/command_test.go:20
  Run
  /Users/jedge/git/simspace/gitlab-merge-request-resource/pkg/in/command_test.go:49
    When it has a minimal valid configuration
    /Users/jedge/git/simspace/gitlab-merge-request-resource/pkg/in/command_test.go:87
      [It] Should clone repository
      /Users/jedge/git/simspace/gitlab-merge-request-resource/pkg/in/command_test.go:89

  Test Panicked
  In [It] at: /usr/local/Cellar/go/1.19.1/libexec/src/runtime/panic.go:260

  runtime error: invalid memory address or nil pointer dereference

  Full Stack Trace
    github.com/samcontesse/gitlab-merge-request-resource/pkg/in.(*Command).Run(0xc000202450, {_, _}, {{{0xc00020c120, 0x2c}, {0x14559e8, 0x1}, 0x0, 0x0, 0x0, ...}, ...})
    	/Users/jedge/git/simspace/gitlab-merge-request-resource/pkg/in/command.go:38 +0xe7
    github.com/samcontesse/gitlab-merge-request-resource/pkg/in_test.glob..func1.3.2.1()
    	/Users/jedge/git/simspace/gitlab-merge-request-resource/pkg/in/command_test.go:101 +0x138
------------------------------


Summarizing 1 Failure:
  [PANICKED!] In Run When it has a minimal valid configuration [It] Should clone repository
  /usr/local/Cellar/go/1.19.1/libexec/src/runtime/panic.go:260

Ran 1 of 1 Specs in 0.003 seconds
FAIL! -- 0 Passed | 1 Failed | 0 Pending | 0 Skipped
--- FAIL: TestIn (0.00s)
FAIL
FAIL	github.com/samcontesse/gitlab-merge-request-resource/pkg/in	0.292s
ok  	github.com/samcontesse/gitlab-merge-request-resource/pkg/out	(cached)
FAIL
```

Using the debugger, I walked through the steps and saw that it appears to be hitting a nil pointer trying to access `user.Email` here:

```
> github.com/samcontesse/gitlab-merge-request-resource/pkg/in.(*Command).Run() ./pkg/in/command.go:38 (PC: 0x147956d)
    33:		if err != nil {
    34:			return Response{}, err
    35:		}
    36:
    37:		user, _, err := command.client.Users.CurrentUser()
=>  38:		err = command.runner.Run("config", "--global", "user.email", user.Email)
    39:		if err != nil {
    40:			return Response{}, err
    41:		}
    42:
    43:		err = command.runner.Run("config", "--global", "user.name", user.Name)
```

I think this is probably related to this commit, adding a feature to set user name and email from the current user: https://github.com/samcontesse/gitlab-merge-request-resource/pull/100

It looks like there needs to be data configured into the user field before testing in order for the `CurrentUser` method to work correctly.

--- a/pkg/in/command_test.go
+++ b/pkg/in/command_test.go
@@ -3,11 +3,6 @@ package in_test
 import (
 	"encoding/json"
 	"fmt"
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-	"github.com/samcontesse/gitlab-merge-request-resource/pkg"
-	"github.com/samcontesse/gitlab-merge-request-resource/pkg/in"
-	"github.com/xanzy/go-gitlab"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
@@ -15,6 +10,12 @@ import (
 	"path/filepath"
 	"strings"
 	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/samcontesse/gitlab-merge-request-resource/pkg"
+	"github.com/samcontesse/gitlab-merge-request-resource/pkg/in"
+	"github.com/xanzy/go-gitlab"
 )
 
 var _ = Describe("In", func() {
@@ -78,6 +79,16 @@ var _ = Describe("In", func() {
 			mux.HandleFunc("/api/v4/projects/42/repository/commits/abc", func(w http.ResponseWriter, r *http.Request) {
 				commit := gitlab.Commit{CommittedDate: &t}
 				output, _ := json.Marshal(commit)
+				w.Header().Set("content-type", "application/json")
+				w.WriteHeader(http.StatusOK)
+				w.Write(output)
+			})
+			mux.HandleFunc("/api/v4/user", func(w http.ResponseWriter, r *http.Request) {
+				user := gitlab.User{
+					Username: "test",
+					Email:    "test@example.com",
+				}
+				output, _ := json.Marshal(user)
 				w.Header().Set("content-type", "application/json")
 				w.WriteHeader(http.StatusOK)
 				w.Write(output)

